src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Yes, with the assembly file name changed and referencing the corelib file (for model.xml and the preprocessed file).